tests/test_uri.ml

```diff
@@ -0,0 +1,28 @@
+let test_v =
+  let check repo_uri expected =
+    let actual =
+      match Dune_release.Github_uri.v repo_uri with
+      | Ok { user; repo } -> Some (user, repo)
+      | Error _ -> None
+    in
+    let ty = Alcotest.(option (pair string string)) in
+    let test () = Alcotest.check ty repo_uri expected actual in
+    let name = "uri: " ^ repo_uri in
+    (name, `Quick, test)
+  in
+  [
+    check "git@github.com:username/repo.git" (Some ("username", "repo"));
+    check "git@github.com:user-name/repo.git" (Some ("user-name", "repo"));
+    check "git@github.com:user-name-123/repo.git"
+      (Some ("user-name-123", "repo"));
+    check "git@github.com:123/repo.git" (Some ("123", "repo"));
+    (* Same as above but without the .git part *)
+    check "git@github.com:username/repo" (Some ("username", "repo"));
+    check "git@github.com:user-name/repo" (Some ("user-name", "repo"));
+    check "git@github.com:user-name-123/repo" (Some ("user-name-123", "repo"));
+    check "git@github.com:123/repo" (Some ("123", "repo"));
+    check "wrong" None;
+    check "https://github.com/username/repo.git" (Some ("username", "repo"));
+  ]
+
+let suite = ("uri", test_v)
```

Comment:
This should be "Github_uri" and the file should be named `test_github_uri.ml` I think!

Reply:
done